training/docs/start/hydra-intro.rst

```diff
@@ -86,3 +86,11 @@ The following missing config options which must be overridden by users:
 -  ``hardware.files.graph``: If you have pre-computed a specific graph,
    specify its filename here. Otherwise, a new graph will be constructed
    on the fly and written to the filename given.
+
+*********************************
+ Validation of the Configuration
+*********************************
+
+The configuration is validated using `Pydantic` before a training run
+starts. To turn this off, you can use the `--no-validation` flag in your
```

Comment:
I think here I'd propose to have a short config example with the `no_validation: True` and `no_validation: False` to explain Pydantic behaviour (happy to help I you'd like)